README.md

```diff
@@ -32,6 +32,8 @@ Bad English &lt;-> Spanish translations.
 - They who don't cry, aren't breastfed.
 - It licks me an egg.
 - Pulled from the hairs
+- You touched dance to the ugliest
```

Comment:
"It touched you dance with the ugliest"?

"You touched" es "vos tocaste", el sujeto en "te tocó" es tácito.